src/cppcache/src/CacheXmlParser.cpp

```diff
@@ -1165,7 +1166,7 @@ void CacheXmlParser::startPersistenceManager(const xmlChar** atts) {
         throw CacheXmlException(s.c_str());
       }
 
-      ACE_OS::strncpy(libraryFunctionName, (char*)atts[i], len);
+      std::strncpy(libraryFunctionName, (char*)atts[i], len);
```

Comment:
Maybe this would be better as a separate pull request for fixing all c-style casts?  However, if you feel it's more effective to update them incrementally as we make changes then I will add them to this pull request.